custom/enikshay/tasks.py

```diff
@@ -163,10 +165,13 @@ def run_batch(self, case_ids):
             updates = []
             for episode in self._get_open_episode_cases(case_ids):
                 did_error = False
+                adherence_cases = None
                 update_json = {}
                 for updater in self.updaters:
                     try:
                         potential_update = updater(self.domain, episode).update_json()
+                        if isinstance(updater, EpisodeAdherenceUpdate):
+                            adherence_cases = updater.get_all_adherence_cases()
```

Comment:
I think we only want to update the ledger values for the "winning" case. I've added some info in the spec.